docs/performance/getting-started.md

```diff
@@ -51,23 +51,27 @@ ngOnInit() {
 }
 ```
 
-### `trace(name:string)`
+### `trace(name, options?)`
 
 The most basic operator, `trace` will measure the amount of time it takes for your observable to either complete or emit its first value. Beyond the basic trace there are several other operators:
 
-### `traceUntil(name:string, test: (T) => Boolean)`
+### `traceUntil(name, test: (T) => Boolean, options?)`
 
 Trace the observable until the first emission that passes the provided test.
 
-### `traceWhile(name:string, test: (T) => Boolean)`
+If the `orComplete` option is passed it will complete the trace when the observable completes, even if an emission never passed the provided test.
 
-Trace the observable until the first emission that fails the provided test.
+### `traceWhile(name, test: (T) => Boolean, options?)`
 
-### `traceUntilLast(name:string)`
+Starting with an emission that passes the provided test, trace until an emission fails the test.
+
+If the `orComplete` option is passed it will complete any existing trace when the observable completes.
```

Comment:
It looks like `orComplete` isn't a field in TraceOptions, but a separate object. Should it be added to TraceOptions since it looks like most of the trace* functions use it?